community/cypher/cypher-compiler-3.3/src/test/scala/org/neo4j/cypher/internal/compiler/v3_3/planner/logical/idp/CartesianProductsOrValueJoinsTest.scala

```diff
@@ -32,7 +32,8 @@ class CartesianProductsOrValueJoinsTest
   val planC = allNodesScan("c")
 
   private def allNodesScan(n: String): LogicalPlan = {
-    val solved = CardinalityEstimation.lift(RegularPlannerQuery(queryGraph = QueryGraph(patternNodes = Set(n))), Cardinality(0))
+    val cardinality = (n.head.toInt - 96) * 1000
```

Comment:
Where do these magic numbers come from?

Reply:
It made sense at the time... I'll add some comments.